core/src/test/java/tech/tablesaw/TableTest.java

```diff
@@ -8,16 +8,23 @@
 import tech.tablesaw.api.Table;
 import tech.tablesaw.columns.Column;
 
+import java.util.Random;
+
 import static org.junit.Assert.*;
+import static org.junit.Assume.assumeFalse;
+import static org.junit.Assume.assumeTrue;
 
 /**
  * Tests for Table
  */
 public class TableTest {
 
+    private static final int ROWS_BOUNDARY = 1000;
+
     private Table table;
```

Comment:
I find it makes the code quite difficult to follow having this be a class member vs having each test create its own instance of a table. All the methods have side effects on this table and it would be much clearer they acted upon the table if they took it as a parameter

Reply:
Methods like `appendColumn` now accept table as a parameter.